### DISCUSSION TITLE
Printer redirection

### DISCUSSION THREAD

**Discussion by komfort64** [leecher1337/ntvdmx64#233] [Q&A]
Hi,
how can I use the redirection of printer ports within the ntvdmx64-ccpu?

**Comment by leecher1337** [comment]
There are some hints in the Issue tracker:

https://github.com/leecher1337/ntvdmx64/issues/161
https://github.com/leecher1337/ntvdmx64/issues/273
https://github.com/leecher1337/ntvdmx64/issues/245

Generally, "net use" redirection should work, too.
